packages/millicast-sdk/src/Publish.js

```diff
@@ -310,10 +309,15 @@ export default class Publish extends BaseWebRTC {
    */
   sendMetadata (message, uuid = DOLBY_SEI_DATA_UUID) {
```

Comment:
Now that I am seeing, message is set as a string, but it could also be an object, etc.

Reply:
can update the definition